app/models/role.rb

```diff
@@ -0,0 +1,10 @@
+class Role < ActiveRecord::Base
+  has_and_belongs_to_many :users, :join_table => :users_roles
```

Comment:
`join_table` should be used by default. But probably it expects name roles_users because `r` is above `u` here https://en.wikipedia.org/wiki/English_alphabet